lib/index.d.ts

```diff
@@ -437,6 +437,11 @@ declare namespace Realm.Sync {
 
     type ErrorCallback = (session: Session, error: SyncError) => void;
     type SSLVerifyCallback = (sslVerifyObject: SSLVerifyObject) => boolean;
+    enum SessionStopPolicy {
```

Comment:
Why make a string enum and not a numeric?

Reply:
Because js doesn't have real enums 😑 This is useable only from typescript projects and people using plain js would be annoyed to use some magic numbers. We can emulate an enum for sure, but I guess it's not very idiomatic.

Reply:
See https://github.com/realm/realm-js/pull/2262#discussion_r262000800